playbooks/common/openshift-cluster/upgrades/docker/upgrade.yml

```diff
@@ -0,0 +1,44 @@
+---
+# We need docker service up to remove all the images, but these services will keep
+# trying to re-start and thus re-pull the images we're trying to delete.
+- name: Stop containerized services
+  service: name={{ item }} state=stopped
+  with_items:
+    - "{{ openshift.common.service_type }}-master"
+    - "{{ openshift.common.service_type }}-master-api"
+    - "{{ openshift.common.service_type }}-master-controllers"
+    - "{{ openshift.common.service_type }}-node"
+    - etcd_container
+    - openvswitch
+  failed_when: false
+  when: openshift.common.is_containerized | bool
```

Comment:
I would like to see if you could put some thought into how we could potentially get this pushed back down into the docker/openshift_docker role at some point. As an interim step, I see us completely needing a separate playbook, but for the long run, I think we will need to incorporate the docker upgrade scenario back into the roles to not sacrifice the pure-ansible usability.